java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -155,6 +151,29 @@ protected void copyFromRunfiles(String path, String destpath) throws IOException
     Files.copy(origin.toPath(), dest.toPath());
   }
 
+  /**
+   * Copy the whole directory from the runfiles under {@code directoryToCopy} to the current workspace.
+   */
+  protected void copyDirectoryFromRunfiles(final String directoryToCopy) throws IOException {
+    File startingDirectory = getRunfile(directoryToCopy);
```

Comment:
Should we handle the case where we get a file and and not a directory?

Reply:
Fixed the wildcard, IDEA is stubborn ;-)